conanfile.py

```diff
@@ -3,14 +3,15 @@
 from conan import ConanFile
 from conan.tools.files import get, replace_in_file, download, rmdir, copy
 from conan.tools.scm import Version
+from conan.tools.gnu import Autotools
```

Comment:
I'd suggest importing this only in the function where it's used since we never need it on Windows. This would match how `AutotoolsToolchain` and `AutotoolsDeps` are imported.